overhaul/lifeboats.md

```diff
@@ -0,0 +1,68 @@
+
+## Content
+
+Everything in the hijack sequence on the Almayer's end should be automatic for both realism and gameplay flow. Sure, it's a dilapidated ship, but even a dilapidated ship would still be able to evacuate its crew to the best of its abilities.
+
+### Timeline
+
+* T+00:00 - Dropship crashes into the Almayer. ARES, communications, and power all shut down or run on emergency.
+* T+00:05 - Lifeboat countdown and evacuation begins automatically.
+* T+01:00 - Individual escape pods are primed for evacuation. May be launched at any time.
+* T+02:00 - Randomized distress beacon automatically called. May be disabled by command(?)
+* T+03:00 - Self-destruct initiates. May be deactivated until T+08:00 by anyone; however this does not prevent the lifeboats from launching. See 'Self-Destruct' section below.
+* T+08:00 - Boarding begins. Lifeboat doors open for Marines (and maybe Xenos) to flood in.
+* T+10:00 - Lifeboat doors seal and try to launch automatically regardless of survivors. Xenos disable this as a failsafe. Grace period begins.
+* T+11:00 - Grace period ends with round-end. If the SD remains primed, it triggers now, incinerating everyone on the ship, remaining humans included.
+
+### Round-end
+
+* Xenos will *always* win once the dropship is hijacked.
+* Activation of the lifeboats (and subsequently SD) results in a xeno minor victory. 
+* Failure of the lifeboats (due to xenos taking over) results in a xeno major.
+* Activation of the lifeboats, but manual deactivation of the SD, still results in a xeno minor, except everyone on the ship isn't killed in a nuclear explosion.
+
+### Self-Destruct
+
+Self-destruct (SD) will be automatically primed at T+03:00 and will gradually arm itself until locking at T+08:00. SD consists of six control rods and a central console; these rods will gradually rotate themselves as the timer progresses before self-detonating at T+11:00.
+
+Anyone can attempt to override the self-destruct and disable it, but this takes time, sped up by engineering skill. The Commanders (CO or XO) should be able to individually instantly disarm the SD system via biometrics (i.e. not their ID card).
+
+SD will be locked in at T+08:00 and cannot be deactivated if it wasn't before then.
+
+## Alternatives
+
+Or maybe it'd be better to just bring old SD itself back with the timer system. Who knows.
+
+## Potential Changes
+
+* For the lifeboats sequence, the individual escape pods could just be axed entirely depending on how much sense they make.
+* Times are subject to change. Dev consensus + numerous test-runs of the lifeboat hold itself show that 10 minutes is a sweet spot for Marines to hold.
+* Larva surge removal and 10-minute hold combined could end up making the timer too short, too. Find out in testmerge.
+* Could also just make SD unable to be overridden by anyone *except* the CO or XO, to prevent abuse.

```

Comment:
Remove any mentions of testmerges or PR. This is a design doc explaining how your feature will work